packages/AdaptiveCards/Microsoft.Bot.Components.AdaptiveCards.csproj

```diff
@@ -0,0 +1,60 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Bot.Components.AdaptiveCard</PackageId>
+    <RepositoryUrl>https://github.com/microsoft/botframework-components/tree/main/packages/AdaptiveCards</RepositoryUrl>
```

Comment:
ReposituryUrl property is included in Directory.Build.props, this can be removed

Reply:
This is the URL composer links to for a components documentation.  I feel like it should point at the specific component so that the user gets the README for how to use that component.  If it points at the root repository they then have to hunt for the usage docs.

---
In reply to: [609116142](https://github.com/microsoft/botframework-components/pull/862#discussion_r609116142) [](ancestors = 609116142)